src/accordion.test.visuals.ts

```diff
@@ -7,7 +7,7 @@ test.describe('Accordion', () => {
 
       const handle = await page.waitForSelector('glide-core-accordion');
 
-      await handle?.evaluate((node) => {
```

Comment:
I swear TypeScript made me do it. Maybe something changed with the upgrade to 5.8.